### DISCUSSION TITLE
Left border colors for opener tabs or nodes

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#135] [Ideas]
I just want to thank you again for realising this massive firefox addon that makes our life easier working with tabs. I really find it helpful your opener tabs feature and I will use it. But if it's not too much trouble for you, I'd be glad if there could be left border colors and indents implemented.
Idea: In the GH settings, new different left border colors with 3 different indents (border-left) could be introduced, where a user can set color and indent options: first color/indent when the tab is only a parent tab, second color/indent when tab is only a node, third color/indent when the tab is both a parent and a node.
For example for the parent tab:
border-left: 0.5rem solid;  
border-left-color: purple;
For only node tab:
border-left: 1rem solid;  
border-left-color: orange;
For tab that is parent and has nodes:
border-left: 1rem solid;  
border-left-color: blue;

I'd be happy if you could implement this.

**Comment by madprops** [comment]
I don't like the idea of another dimension in tab styling. Parent/nodes are not a major concept of GH, and I don't want to steer it into that. Also, there's already icons for both tabs with nodes and tabs with parents that provide visual feedback. The icons can be placed left or right.

**Reply by N3C2L** [reply]
Ok, I totally get that. I don't want to cause too much trouble with this parent/nodes feature I promise. It would be just the only thing I had in my mind regarding parent/nodes where I could better visually navigate in the item list if I see where that left border color / indent  on a parent or node is. Advantage for me would be also, that the icon list would be less crowded if I use the left borders or indents instead of the icons.

**Reply by N3C2L** [reply]
It doesn't need to be on by default this left border colors, it would be just nice to have, if it's ok.

**Reply by N3C2L** [reply]
Ok, I wrote that I don't want to make problems with this idea, so I'll close this thread. 👍
